tests/CMakeLists.txt

```diff
@@ -29,6 +29,22 @@ file(GLOB_RECURSE TEST_SOURCES
         ${CMAKE_SOURCE_DIR}/tests/conftest.cpp
         ${CMAKE_SOURCE_DIR}/tests/utils.cpp
```

Comment:
Could we also glob for any `.h, .cpp` to be added instead (unless you want to separate `test_*.cpp` somehow to enforce test name pattern)?

Same for `${CMAKE_SOURCE_DIR}/tests/*/platf/windows/` level. I know that I will have some mocks in `${CMAKE_SOURCE_DIR}/tests/*/platf/windows/` that will belong to windows only.

Reply:
I don't think `.h` is needed here. I know we do that in Sunshine, but I think that was a mistake. It should be covered by `include_directories("${CMAKE_SOURCE_DIR}")`

I would like to enforce a naming pattern just for consistency, but they don't necessarily all need to start with `test_`. Is there a pattern you are predicting to use? Maybe `setup_` or `mock_`?

Reply:
I would be fine with `mock_`.